app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split(" ")
```

Comment:
better to use unpacking and save the parts into variables with meaningful names. files[1] , files[2] are not informative